include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -94,10 +94,10 @@ struct __is_heap_check;
 template <typename _Predicate, typename _ValueType>
 struct __create_mask_unique_copy;
 
-template <class _Tag, typename _ExecutionPolicy, typename _Tp, typename>
+template <class _Tag, typename _Tp, typename>
```

Comment:
To remove `, typename`.

Reply:
Fixed

Reply:
I don't know from implementation point of view how to do this.
@MikeDvorskiy could you please prepare some separate PR with this fix?